src/Components/CreateImageWizard/steps/Users/component/Empty.tsx

```diff
@@ -10,19 +10,23 @@ import {
 } from '@patternfly/react-core';
 import UserIcon from '@patternfly/react-icons/dist/esm/icons/user-icon';
 
-const EmptyUserState = () => {
+interface EmptyUserStateProps {
+  onAddUserClick: () => void;
+}
+const EmptyUserState = ({ onAddUserClick }: EmptyUserStateProps) => {
   return (
     <EmptyState variant={EmptyStateVariant.lg}>
       <EmptyStateHeader
         icon={<EmptyStateIcon icon={UserIcon} />}
         headingLevel="h4"
       />
       <EmptyStateFooter>
-        <Button variant="secondary" onClick={() => {}}>
+        <Button variant="secondary" onClick={onAddUserClick}>
```

Comment:
Switching between the empty and user creation states could be handled in a similar way like FSC I think, we could introduce a helper into the users structure called `userMode` or smth like that and then switch what renders on the step based on a store value.

But this might also be a follow up.